src/main/java/org/json/JSONObject.java

```diff
@@ -2809,8 +2819,6 @@ static final Writer writeValue(Writer writer, Object value,
             }
         } else if (value instanceof Boolean) {
             writer.write(value.toString());
-        } else if (value instanceof Enum<?>) {
```

Comment:
Please revert this change. I believe the code works correctly with the Enum case in the original location.

Reply:
yes, it does. Done.